src/test/java/com/amazon/sqs/javamessaging/SQSSessionTest.java

```diff
@@ -1482,19 +1284,14 @@ public void testCreateObjectMessage() throws JMSException, InterruptedException
      * Test create byte message
      */
     @Test
-    public void testCreateTextMessage() throws JMSException, InterruptedException {
+    public void testCreateTextMessage() throws JMSException {
 
         sqsSession.setClosed(true);
 
         /*
          * Create bytes message
          */
-        try {
-            sqsSession.createTextMessage();
-            fail();
-        } catch(IllegalStateException ise) {
-            assertEquals("Session is closed", ise.getMessage());
-        }
+        assertThrows(IllegalStateException.class, () -> sqsSession.createTextMessage(), "Session is closed");
 
         sqsSession.setClosed(false);
```

Comment:
Wow, that was a huge test, may be we should consider splitting the file in the future for clarity.

Reply:
Will do. Need to do an assessment first. thanks for the suggestion.